README.md

```diff
@@ -58,9 +58,49 @@ Do either of the following:
 -   Move them to `/usr/local/lib` or `/usr/lib`.
 -   Set the `LIBRARY_PATH` environment variable to the directory containing the libraries.
 
-### Execution
+### Compilation (static libraries)
+
+For iOS development you have to use static libraries. Get the static libraries from the [vosk-api][vosk-api-ios] team.
+
+#### macOS-only
```

Comment:
This would be better written as a bullet point list for conciseness.

- Create a non-fat file from libvosk.a
\<example code\>
- [Mark your crate type as `staticlib`](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#the-crate-type-field)
- Create a [build script](https://doc.rust-lang.org/cargo/reference/build-scripts.html) and provide cargo with the path to the libraries with `cargo:rustc-link-search=static=` or `cargo:rustc-link-lib=static=`

Of course these are just examples, I'm not experienced with iOS development, so you're free to add more info to each point.